Add zh_CN i18n Support

NPBeta opened the pull request:
Add Chinese (zh-CN) language file for built-in language support.

Trivernis committed:
Update README.md

Trivernis committed:
Update bug_report.md

Trivernis committed:
Merge pull request #71 from Trivernis/develop

Version 1.2.0

Trivernis committed:
Update bug_report.md

People just don't report the exact server version!

Trivernis committed:
Merge pull request #72 from Trivernis/develop

Fix #66, #70 Task progress being discarded on resume

NPBeta committed:
Add zh_CN i18n Support

Trivernis commented:
I've asked some people to look over it, just to make sure it's correct

Trivernis reviewed (changes requested):
@NPBeta The results of the review by @ed3d3d (twitter handle) I've requested are:

**TASK_SAVE_FAILED:**
switch 错误 to 失败
保存任务时发生错误 ---> 保存任务失败

**TP_ONLY_PLAYER:** Missing Translation
Suggestion: 只有玩家可用这个命令

NPBeta committed:
Added Missing Translation & Tweak Translation

Fix unconsistent translation:
Using 失败 for errors that has no further information.
Using 发生错误 for exceptions with stacktrace or other informations.

NPBeta commented:
> switch 错误 to 失败
> 保存任务时发生错误 ---> 保存任务失败

I've checked the source text, and decided to make the translation more consistent as following:
For `Failed to xxx`, `Error when xxx`,etc. :
  - Using `xxx 失败`, because errors like this will not provide error details.
For `Exception when xxx: (details)`,etc. :
  - Using `xxx 时发生错误: (details)` to provide`错误`'s details.
